Jenkinsfile

```diff
@@ -369,15 +369,8 @@ try {
                 }
               }
               stage('sign') {
-                def buildType = bat (
-                  script: "@type BUILDTYPE",
-                  returnStdout: true
-                ).trim().toLowerCase()
-
-                def packageName = "RStudio-${buildType}-${rstudioVersionMajor}.${rstudioVersionMinor}.${rstudioVersionPatch}-RelWithDebInfo"
-                if (buildType == "release") {
-                  packageName = "RStudio-${rstudioVersionMajor}.${rstudioVersionMinor}.${rstudioVersionPatch}-RelWithDebInfo"
-                }
+
+                def packageName = "RStudio-${rstudioVersionMajor}.${rstudioVersionMinor}.${rstudioVersionPatch}"
```

Comment:
Doesn't this still need RelWithDebInfo since that's the filename produced by cpack? (we don't rename it until upload)

Reply:
Fixed